### DISCUSSION TITLE
Fifths?

### DISCUSSION THREAD

**Discussion by katiereiter** [rxhanson/RectanglePro-Community#174] [Q&A] [answered]
Is there any way to break a screen into fifths?

**Comment by rxhanson** [comment, marked as answer]
Yes, you can create Custom Shortcuts for this:

![image](https://user-images.githubusercontent.com/13651296/202541743-9b4ab9c7-47e8-431d-b127-f4cdc4941975.png)

**Reply by katiereiter** [reply]
Amazing, thank you!!
